Android.mk

```diff
@@ -52,13 +52,15 @@ ifeq "$(HYBRIS_FSTABS)" ""
 TARGET_VENDOR := "$(shell echo $(PRODUCT_MANUFACTURER) | tr '[:upper:]' '[:lower:]')"
 HYBRIS_FSTABS := $(shell find device/$(TARGET_VENDOR) -name *fstab* | grep -v goldfish)
 endif
-# Some devices devices have the short vendor name in PRODUCT_BRAND so try to
+# Some devices have the short vendor name in PRODUCT_BRAND so try to
 # search from device/PRODUCT_BRAND if fstab files are still not found.
 ifeq "$(HYBRIS_FSTABS)" ""
 TARGET_VENDOR := "$(shell echo $(PRODUCT_BRAND) | tr '[:upper:]' '[:lower:]')"
 HYBRIS_FSTABS := $(shell find device/$(TARGET_VENDOR) -name *fstab* | grep -v goldfish)
```

Comment:
Wouldn't this find the fstab.qcom? Can you provide a full fstab path?

Reply:
The part above my own changes, doens't find anything with "PRODUCT_BRAND"
my fstab.qcom file is in "device/xiaomi/msm8996-common/rootdir/etc/fstab.qcom"